api/envoy/extensions/clusters/dynamic_forward_proxy/v3/cluster.proto

```diff
@@ -53,4 +56,21 @@ message ClusterConfig {
   //   <https://datatracker.ietf.org/doc/html/rfc7540#section-9.1.1>`_
   //
   bool allow_coalesced_connections = 3;
+
+  // Enable strict dns sub cluster instead logic dns.
+  SubClustersConfig sub_clusters_config = 4;
+}
+
+message SubClustersConfig {
+  // The maximum number of sub clusters that the DFP cluster will hold. If not specified defaults to 1024.
+  google.protobuf.UInt32Value max_sub_clusters = 1 [(validate.rules).uint32 = {gt: 0}];
```

Comment:
Can't this be controlled by the existing circuit breaker configuration? Why do we need a new value here?